tools/datasets/tum/download_tum.sh

```
@@ -0,0 +1,63 @@
+        -*) echo "error: unknown option '$1'" >&2; exit 2 ;;
+        *)  out_dir="$1"; shift ;;
+    esac
+done
+
+readonly -a urls=(
+    "https://cvg.cit.tum.de/rgbd/dataset/freiburg3/rgbd_dataset_freiburg3_long_office_household.tgz"
+)
+
+download_file() {
+    local url="$1"
+    local dest="${out_dir}/$(basename -- "${url}")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)